okdata/cli/commands/datasets/boilerplate/config.py

```diff
@@ -127,6 +140,7 @@ def boilerplate_prompt(include_extra_metadata=True):
             "name": "pipeline",
             "message": "Prosessering",
             "choices": pipeline_choices,
+            "when": lambda x: x["sourceType"] == "file",
```

Comment:
Also for `event`?

Reply:
Requiers a different set of pipeline choices (e.g. event-copy, none)

Reply:
Sure!

However `event-transform` doesn't give any value out of the box, right? So maybe offer `event-copy` and `none` for now, and still require manual configuration for setting up transformations?